### ISSUE TITLE
How to build hat bundle?

### ISSUE THREAD

**Issue by c2v4** [eDonnes124/Town-Of-Us-R#328]
In Readme there is a lot of info about the game, but not so much about the project itself. I was able to build the project with `dotnet.build` but  still - how to build files `touhats.bundle` and `touhats.catalog` from graphics?

**Comment by whichtwix** [comment]
currently only the devs are able to make those, the method to add your own hats might come in the future

**Comment by c2v4** [comment]
Is that because it's difficult to do, or you don't want to reveal it now?

**Comment by whichtwix** [comment]
well the dev who originally made it was planning to release the method, but I think he got busy so dont know if he will get to it eventually

**Comment by eDonnes124** [comment]
Yeah I didn't make the hat bundles. I stopped modding for about a year when this all got changed and I don't even make it so I can't provide much input on this. Also I'm pretty sure some of the hat sizes all got messed up when porting to the new version as well.

**Comment by whichtwix** [comment]
@c2v4 please join the tour discord server and make a ticket from #tour support, mydragonbreath will provide you with the instructions there

**Comment by MalkizH3** [comment]
I’d love to make a small custom mod just for me and my friends, and I was wondering — would you be willing to share the Unity project you used to create the touhats.catalog and .bundle files?
